ova/Libraries/automatic_set_ram.sh

```diff
@@ -0,0 +1,10 @@
+etc_elastic="/etc/elasticsearch"
+ram_gb=$(free -g | awk '/^Mem:/{print $2}')
```

Comment:
This should go in megabytes so you should use `free -m`

Reply:
Done in 71fd88cf61ad859245d50e315178f0058cb61754